src/main/java/core/basesyntax/service/operations/SupplyOperation.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.model.FruitTransaction;
+
+public class SupplyOperation implements OperationHandler {
```

Comment:
The `SupplyOperation` class should be in the `strategy` package according to the project structure defined in the checklist.